.github/workflows/integration.yaml

```diff
@@ -35,6 +35,14 @@ jobs:
             exit 1
           fi
 
+      - name: Run go mod vendor
+        run: |
+          go mod vendor
+          if ! git diff --quiet; then
+            echo "Please run 'go mod vendor' to make vendored copy of dependencies"
```

Comment:
go mod tidy && go mod vendor

Reply:
@cheyang There already is a step for checking `go mod tidy`, see: https://github.com/kubeflow/arena/blob/29b2d6d2c5df6959a05c9de8a81c7436a7820d98/.github/workflows/integration.yaml#L30-L36

Reply:
Ok